pythnet-price-feeds/cosmwasm.md

```diff
@@ -6,9 +6,30 @@ description: Consume Pyth Network prices in applications on Cosmwasm
 
 Cosmwasm contracts can update and fetch the Pyth prices using the Pyth Cosmwasm Contract, deployed on their network. The documented source code can be found [here](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/cosmwasm/contracts/pyth).
 
-## Updating Price Feeds
+## Price Feeds
+Pythnet price feeds uses an on-demand price update model, where users are responsible for posting price updates on-chain when needed. The latest price can be fetched from a price service, which is a permissionless centralised service. We recommend protocols to run their own price-service. 
+Pythnet publishes prices for two kinds of feeds: 
+1. Stable price feeds are the most reliable price feeds.
+2. Edge price feeds comes with the latest and greatest features of Pyth price feeds as soon as they are available. 
```

Comment:
I think these items need a little more technical explanation so the reader understands why these are different. Something like:

1. Stable price feeds consist of prices generated on the pythnet blockchain. These feeds are available on both mainnet and testnet blockchains. Use stable feeds if you would like your test environment to be identical to your production environment. 
2. Edge price feeds consist of prices generated on the pythtest blockchain, which is Pyth's test environment for new features. Consequently, these feeds are not as reliable as the stable feeds, and there are other differences as well (e.g., different price feed ids). These feeds are only available on testnets.

Most users should pick stable feeds, unless you are explicitly trying to test out a new feature that hasn't launched yet.